source/interpr.d

```diff
@@ -71,6 +76,11 @@ int main (string [] args)
 		return 0;
 	}
 
+	if(firstString.startsWith("#unroll")){

```

Comment:
So, the way to invoke this is to say `#unroll <somenumber>` on the first line. In a finished product, people should know this without looking at the code. OK for now though.